projects/kit/components/pager/pager.styles.less

```diff
@@ -42,22 +68,33 @@
         &:not(._visible)::before,
         &:not(._visible) + .t-item:not(._active):not(._pre-active):empty::before,
         &:not(._last):not(._post-active):not(._active):nth-last-child(1 of ._visible)::before {
-            transform: scale(0.25) !important;
-
-            :host-context(tui-root._mobile) & {
-                transform: scale(0.333) !important;
-            }
+            transform: var(--t-super-small-dot) !important;
         }
 
         &:not(._last)._post-active:nth-last-child(1 of ._visible)::before,
         &:not(._visible) + .t-item._pre-active:empty::before,
         &:not(._visible) + .t-item:empty + .t-item:not(._active):empty::before,
         &:not(._pre-last):nth-last-child(2 of ._visible):not(._active)::before {
-            transform: scale(0.5);
+            transform: var(--t-small-dot);
+        }
 
-            :host-context(tui-root._mobile) & {
-                transform: scale(0.666);
-            }
+        & + .t-item:not(:empty)._pre-last {
+            transform: scale(0.64);
         }
+
+        & + .t-item:not(:empty)._pre-last:nth-last-child(2 of ._visible) {
+            transform: scale(1);
+        }
+    }
+
+    &:not(:empty)::ng-deep > tui-icon {
```

Comment:
No need for :not(:empty) if it has tui-icon

Reply:
fixed